### PR TITLE
chore(docs): Add google auth hook guide

### PR DESCRIPTION
**Pull request by VictorJSV**
I added Google auth hook guide :)

### CODE REVIEW

File: docs/guides/google-auth-hook.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+}
+
+unleash.start(options).then(unleash => {
+  console.log(`Unleash started on http://localhost:${unleash.app.get('port')}`);
+});;
+```
+
+### Creating a web application client ID
+
+1. Go to the credentials section in the [Google Cloud Platform Console](https://console.cloud.google.com/apis/credentials?_ga=2.77615956.-1991581217.1542834301).
```
Comment (ivarconr) [inline docs/guides/google-auth-hook.md]: LOW: You should probably remove those `_ga` attributes.

**Comment by ivarconr** [comment]
Should we also add some details on how to limit access to certain users? 

E.g.
"Company A would only like their employees to access Unleash."

**Comment by VictorJSV** [comment]
Thanks! I'll prepare another pull request to fix some details and add how to limit access

File: docs/guides/google-auth-hook.md (hunk 1 above)
Comment (VictorJSV) [reply docs/guides/google-auth-hook.md]: Ups! :sweat_smile:

**Pull request merged**